collector/netdev_linux.go

```diff
@@ -50,12 +50,14 @@ func parseNetDevStats(r io.Reader, ignore *regexp.Regexp) (map[string]map[string
 			scanner.Text())
 	}
 
-	header := strings.Fields(parts[1])
+        header_recieve := strings.Fields(parts[1])
```

Comment:
Please use go-style variable names.  For example this should be `recieveHeader`.

Reply:
It's also receive, not recieve